13-tool-usage-cycle-tracking/README.md

```diff
@@ -246,6 +242,34 @@ These environment variables are set in Notehub, either per-device, per-fleet or
 
 Please see our tutorial [Understanding Environment Variables](https://dev.blues.io/guides-and-tutorials/notecard-guides/understanding-environment-variables/) for a fuller description of how to set environment variables in Notehub.
 
+
+### Vibration Monitoring and Alerts
+
+The Notecard accelerometer is used to sense vibration. Tool usage can be inferred from the measured vibration using environment variables that define the expected vibration characteristics. The vibration environment variables define thresholds of vibration for when the tool is inactive and active.
+
+* `alert_off_vibration`: the *maximum* vibration expected when the tool is *inactive*. Minimum value is `0`, which is also the default. Must be less than or equal to `alert_under_vibration`.
+
+* `alert_under_vibration`: the *minimum* vibration expected when the tool is *active*. Must be equal or greater than `alert_off_vibration` and less than `alert_over_vibration`. The default value is `0`.
+
+* `alert_over_vibration`: the *maximum* vibration expected when the tool is *active*. Must be greater than `alert_under_vibration`.
+
+<a name="vibration-category"></a>
+With these variables defined, the measured vibration is classified into into one these vibration categories:
+
+* `none`: when vibration is less than or equal to `alert_off_vibration`
+
+* `low`: when vibration is between `alert_off_vibration` and `alert_under_vibration`
+
+* `normal`: when vibration is between `alert_under_vibration` and `alert_over_vibration`
+
+* `high`: when vibration is greater than `alert_over_vibration`
+
+The `vibration` property is added to power events with the values `none`, `low`, `normal`, `high` corresponding to the vibration categories above. The raw vibration value is also included, as `vibration_raw: 123.45`.
+
+When `alert_off_vibration`, `alert_under_vibration` and `alert_over_vibration` are all undefined or `0`, vibration is not sensed. Otherwise, vibration is sensed and vibration data is included with power monitoring events and alerts.
+
+An alert is sent when the vibration is classified as `low` or `high` which indicates vibration is outside of expected operating parameters. When a vibration alert is triggered, the `alert` property includes the word `vibration`, indicating the alert happened because of too little or too much vibration.
```

Comment:
> An alert is sent when the vibration is classified as `low` or `high` which indicates vibration is outside of expected operating parameters.

An alert is sent when the vibration is classified as `low` or `high`, which indicates vibration is outside of expected operating parameters.